modules/language-guide/pages/sharing.adoc

```diff
@@ -1,12 +1,228 @@
-= Sharing among distinct actors
+= Sharing data and behavior
 
-CAUTION: Not complete
+Recall that in {proglang}, mutable state is always private to an actor.
 
-== Shared data
+However, two actors can share message data, and those messages can
+refer to actors, including themselves and one another.  Additionally,
+messages may refer to individual functions, if those functions are `shared`.
 
-== Shared functions
+Through these mechanisms, two actors can coordinate their behavior
+through asynchronous message passing.
 
-[[example-notify]]
-== Example: Notification service
+== Publisher-subscriber pattern with actors
 
-CAUTION: Not complete (todo: simple example of a callback-using service)
+We show how actors share their functions by focusing on variations of the link:https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern["publisher-subscriber" pattern],
```

Comment:
```suggestion
We demonstrate how actors may share their functions by focusing on variations of the link:https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern["publish-subscribe" pattern] (of contracted to  "pubsub"),
```

Reply:
"actors can share" instead of may
(referred to as "pubsub")